src/python-fastui/fastui/json_schema.py

```diff
@@ -259,16 +260,29 @@ def special_string_field(
             )
         elif enum := schema.get('enum'):
             enum_labels = schema.get('enum_labels', {})
-            return FormFieldSelect(
-                name=name,
-                title=title,
-                placeholder=schema.get('placeholder'),
-                required=required,
-                multiple=multiple,
-                options=[SelectOption(value=v, label=enum_labels.get(v) or as_title(v)) for v in enum],
-                initial=schema.get('default'),
-                description=schema.get('description'),
-            )
+            options = [SelectOption(value=v, label=enum_labels.get(v) or as_title(v)) for v in enum]
+            if schema.get('mode') == 'radio' and multiple:
```

Comment:
addding this currently breaks the demo - as it appears that the `json_schema` param leak as well to the next component (which is a `multiple`)
i think similar to https://github.com/pydantic/FastUI/issues/178